src/corelocation.cs

```diff
@@ -171,7 +171,7 @@ partial interface CLLocationManager {
 		[Protocolize]
 		CLLocationManagerDelegate Delegate { get; set;  }
 
-		[Export ("delegate", ArgumentSemantic.Assign)][NullAllowed]
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Assign)]
```

Comment:
minor: the `[NullAllowed]` was already present here
as much as possible we try not to change code (e.g. formatting) when doing PR so GIT history keeps being useful

Reply:
makes sense. I had mixed feelings about this change as well. I'll revert and rebase.